ensembl/htdocs/info/genome/variation/prediction/protein_function.html

```diff
@@ -23,28 +23,87 @@
 protein function. We compute predictions for each of these tools for all possible 
 single amino acid substitutions in the Ensembl human proteome. This enables us to provide 
 predictions for novel mutations for VEP and API users. We were able to compute predictions
 from at least one tool for over 95% of the human proteins in Ensembl. 
+<br> SIFT predictions are also available for cat, chicken, cow, dog, goat, horse, mouse, pig, rat, sheep and zebrafish.
+</p>
+
+<p>
+Pathogenicity scores from <a href="#REVEL">REVEL</a>, <a href="#MutationAssessor"> MutationAssessor</a> 
+and <a href="#MetaLR">MetaLR</a> are also available for human missense variants.
+To improve compatibility with other resources, these scores are taken from dbNSFP (version 3.5.a). 
 </p>
 
 <p>
-SIFT predictions are also available for cat, chicken, cow, dog, goat, horse, mouse, pig, rat, sheep and zebrafish.
+We provide a brief description of each of these tools and how they are made available in Ensembl. 
+For more detail please see the representative papers listed <a href="#references">below</a>, 
+and the relevant publications available on each tool's website. 
 </p>
 
+<h2>Genome wide scores</h2>
+
+The level of conservation at a locus is strong predictor of the deleteriousness of any change. As an 
+indicator change tolerance, we display GERP conservation scores on our variant pages. 
+CADD scores are also available for human single nucleotide substitutions.
+<p>
+<div style="border:1px solid #CCC;margin-bottom:10px">
+  <div style="padding: 6px 6px">
+    <h2 id="GERP">GERP</h2>
+
 <p>
-Pathogenicity scores from <a href="#CADD">CADD</a>, <a href="#REVEL">REVEL</a>,
-<a href="#MutationAssessor"> MutationAssessor</a> and <a href="#MetaLR">MetaLR</a> are also available for human variants.
-To improve compatibility with other resources we use CADD scores from the CADD (version v1.4) project 
-while the other three are provided by dbNSFP ( version 3.5.a). 
-We display colour-coded scores for each algorithm to support immediate assessment of a set of 
-variants, but recommend the actual scores are taken into account when interpreting the impact 
-of an individual variant. This is especially important for CADD and REVEL which do not have 
+We display conservation scores as computed with the Genomic Evolutionary Rate Profiling 
+<a rel="external" href = "http://mendel.stanford.edu/SidowLab/downloads/gerp/">GERP </a> software
+on <a href = "../../../../info/docs/compara/analyses.html"> Multiple Sequence Alignments </a> of whole-genomes.
+GERP identifies constrained loci in multiple sequence alignments by comparing the level of substitution 
+observed to that expected if there was no functional constraint. 
+Positive scores represent highly-conserved positions while negative scores represent highly-variable positions.
+<li> the highest score of any base in a multi-base deletion is displayed
+<li> the mean of the scores of the two flanking bases is shown for an insertion 
```

Comment:
It is the highest of the two flanking bases that we choose. I'm not sure that we would respect how GERP scores are calculated for a specific position in the genome if we'd take the mean. But I can confirm with compara. It will only require a change in our API.

Reply:
OK. Let's go with the mean then. I will update the API code accordingly.